README.md

```diff
@@ -78,22 +78,8 @@ Each of the *I-Frames* in the resulting AVI file is then replaced with the next
 Finally, the moshed AVI file is "baked", which means it's converted back into an *MP4 file*,
 persisting the artifacts in the AVI file into a stable video file.
 
-# Building from source
-Thanks to golang's flawed dependency system, setting up *moshpit* locally for development
-is a bit of a hassle.
-You can't use `go get`, as we're using modified versions (forks) of some libraries.
-Therefore, you need to clone the repository into your *GOPATH* and use [`go dep`](https://github.com/golang/dep)
-to install dependencies into the `vendor` directory:
+## Building from source
 
-```
-mkdir -p $GOPATH/src/github.com/crushedpixel
-cd $GOPATH/src/github.com/crushedpixel
-git clone https://github.com/CrushedPixel/moshpit
-dep ensure
-``` 
-
-You should now be able to build the `moshpit` command binary:
-
-```
-go build github.com/crushedpixel/moshpit/cmd/moshpit
+```shell
+go get github.com/CrushedPixel/moshpit/cmd/moshpit
```

Comment:
Does this really build from source, or only download the precompiled dependency?
Other than that, changes LGTM

Reply:
It does build from source. Go has no way of knowing where compiled binaries are, but when a URL is specified, it will find the git repo and clone it, then build it if possible.